package.json

```diff
@@ -84,7 +85,7 @@
     "http-server": "^0.12.0",
     "qunit": "^2.9.3",
     "rollup": "^1.28.0",
-    "rollup-plugin-buble": "^0.19.8",
+    "rollup-plugin-babel": "^5.0.0-alpha.1",
```

Comment:
Do we need something in the 'alpha' version here, or would a stable version be better?

Reply:
Well, I'm actually maintainer of that plugin 😅 We are in the process of migrating it into the rollup plugins monorepo - so the final v5 should get released this month. There really won't be any breaking changes for you here. This one is preferable over v4 because it's slightly faster and we can use `skipPreflightCheck` option to avoid unnecessary work (which conflicts a little bit with this custom class transformer introduced in this PR).